static/js/drafts.js

```diff
@@ -140,8 +140,9 @@ exports.restore_draft = function (draft_id) {
                              {select_first_unread: true, trigger: "restore draft"});
         }
     }
-
-    overlays.close_overlay("drafts");
+    if (!is_overlay_closed) {
```

Comment:
We now have a function in `overlays.js` that checks if the drafts overlay is open (`overlays.drafts_open()`), which should be used instead of this boolean.